components/autofill/sql/create_shared_schema.sql

```diff
@@ -22,7 +22,11 @@ CREATE TABLE IF NOT EXISTS addresses_data (
     time_last_modified  INTEGER NOT NULL,
     times_used          INTEGER NOT NULL,
 
-    sync_change_counter INTEGER NOT NULL
+    sync_change_counter INTEGER NOT NULL,
+
+    -- fields on incoming records we don't know about and roundtrip.
+    -- a serde_json::Value::Object as an encrypted string.
```

Comment:
I don't think it will be encrypted for addresses, right? But TBH I don't think we need this column at all - we can just extract it from the mirror table's payload.